src/SFA.DAS.EmployerAccounts.Web/Extensions/HtmlHelperExtensions.cs

```diff
@@ -132,5 +133,17 @@ public static IFooterViewModel GetFooterViewModel(this HtmlHelper html, bool use
             useLegacyStyles: useLegacyStyles
             );
         }
+
+        public static MvcHtmlString GetClientContentByType(this HtmlHelper html, string type, string clientId)
```

Comment:
Do we need to pass in the client Id since we could resolved it from the config?

Reply:
This is the eg: format ?type=banner&clientId=eas-acc_legacy

Reply:
I think we could store this in config. The clientId will remain unchanged for each calling application